published/003-2016-Jan-12.md

```diff
@@ -0,0 +1,127 @@
+# IPFS Weekly #3
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+This is a double weekly: here are some of the highlights for the [January 12th](//github.com/ipfs/pm/issues/79) and the [January 19th](https://github.com/ipfs/pm/issues/83) sprints:
+
+## Updates
+
+* [**ArcticJS**](http://arcticjs.club) [@diasdavid](//github.com/diasdavid), [@whyrusleeping](//github.com/whyrusleeping), [@RichardLitt](//github.com/RichardLitt), and [@noffle](//github.com/noffle) were all able to meet up for the first time as a group and hack together at [ArcticJS](http://arcticjs.club), the first JavaScript conference above the Arctic circle, in Svalbard. Around 15 attended, and there was a lot of hacking, snow, talking about buffers, and swimming in the ocean at midnight underneath the Aurora Borealis (no, really). Most of the work done at the conference was conversations, but included [@diasdavid](//github.com/diasdavid) and [@RichardLitt](//github.com/RichardLitt) working on the API together, [@whyrusleeping](//github.com/whyrusleeping) learning how to javascript, and [@diasdavid](//github.com/diasdavid) and [@xicombd](//github.com/xicombd) working on IPFS together. All in all, it was an incredible trip. The videos for the actual talks (given in the most Northernmost church in the world) are [here](https://www.youtube.com/playlist?list=PL3bvPCw5QCLLJUL2Q_bBI1bi9bYQ-4hci).
+* [**dist.ipfs.io**](https://dist.ipfs.io) A distributions page has been shipped! This was largely spearheaded by [@dignifiedquire](//github.com/dignifiedquire). http://dist.ipfs.io/
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 0.3.11 is shipped! Also, we merged 0.4.0 into master. [@jbenet](//github.com/jbenet), [@Dignifiedquire](//github.com/Dignifiedquire) and others [fixed appveyor](//github.com/ipfs/go-ipfs/pull/2137), which means that windows CI now works. And finally the latest webui by [@Dignifiedquire](//github.com/Dignifiedquire) was pushed in 0.3.11. Check out the [changelog](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md).
```

Comment:
0.4.0 means much more than "windows CI now works". i'd just remove that part, or attempt at describing 0.4.0's main improvements. (the windows CI things is a tiny factor comparatively)